common/postgresql/values.yaml

```diff
@@ -46,3 +46,19 @@ resources:
   requests:
     memory: 256Mi
     cpu: 100m
+
+backup:
+  enabled: false
+  repository: sapcc/backup-tools
+  image_version: v0.4.6
+  interval_full: 3 days
+  interval_incr: 3 days
```

Comment:
Is it correct that these values are both set to 3 days?
Does this mean we only do an incremental backup every 3 days?